### PR TITLE
Javadoc + Asset Refactoring

### PR DESCRIPTION
**Pull request by viksharma-mtv**
This Pull Request covers renaming of Assets (Not specific to dino or robot models) and adds Javadoc comments where necessary.

### CODE REVIEW

File: mediapipe/examples/android/src/java/com/google/mediapipe/apps/instantmotiontracking/Sticker.java [hunk 1]
```diff
@@ -41,99 +52,138 @@ public Render iterate() {
...
-  // Used to determine next stickerID
+  // Used to determine next stickerId
   private static int globalIDLimit = 1;
 
+  /**
+   * Used to create a Sticker object with a newly generated stickerId and a
+   * default Render of the first possible render in our Render enum.
+   */
   public Sticker() {
     // Every sticker will have a default render of the robot animation
```
Comment (tyrmullen) [inline mediapipe/examples/android/src/java/com/google/mediapipe/apps/instantmotiontracking/Sticker.java:74]: nit: remove if we're getting rid of all robot references

File: mediapipe/examples/android/src/java/com/google/mediapipe/apps/instantmotiontracking/BUILD [hunk 2]
```diff
@@ -47,10 +47,10 @@ android_binary(
     name = "instantmotiontracking",
     assets = [
         "//mediapipe/graphs/instantmotiontracking:mobile.binarypb",
-        "//mediapipe/examples/android/src/java/com/google/mediapipe/apps/instantmotiontracking/assets/robot:robot.obj.uuu",
-        "//mediapipe/examples/android/src/java/com/google/mediapipe/apps/instantmotiontracking/assets/robot:robot_texture.bmp",
-        "//mediapipe/examples/android/src/java/com/google/mediapipe/apps/instantmotiontracking/assets/dino:dino.obj.uuu",
-        "//mediapipe/examples/android/src/java/com/google/mediapipe/apps/instantmotiontracking/assets/dino:dino_texture.bmp",
+        "//mediapipe/examples/android/src/java/com/google/mediapipe/apps/instantmotiontracking/assets/asset1:asset_1.obj.uuu",
```
[the remark this answers is not in the document]
Comment (jianingwei) [reply mediapipe/examples/android/src/java/com/google/mediapipe/apps/instantmotiontracking/BUILD]: Agree with Tyler. The actual asset can be named robot in assets/ directory. In the code, you just refer to it as asset_3d or something, which doesn't specify the name of it.

File: mediapipe/examples/android/src/java/com/google/mediapipe/apps/instantmotiontracking/Sticker.java [hunk 3]
```diff
@@ -14,25 +14,35 @@
 
 package com.google.mediapipe.apps.instantmotiontracking;
 
-import java.util.ArrayList;
 import com.google.protobuf.*;
```
Comment (jianingwei) [inline mediapipe/examples/android/src/java/com/google/mediapipe/apps/instantmotiontracking/Sticker.java:17]: This is not encouraged. Can you avoid the * and specify which one you are importing?

File: mediapipe/examples/android/src/java/com/google/mediapipe/apps/instantmotiontracking/Sticker.java [hunk 4]
```diff
@@ -14,25 +14,35 @@
...
+ */
 public class Sticker {
 
+  /** All types of possible objects to render for our application. */
   public enum Render {
     // Every possible render for a sticker object
-    ROBOT, DINO, GIF;
-    // Loop through each possible render object
+    GIF,
+    ASSET_1;
```
Comment (jianingwei) [inline mediapipe/examples/android/src/java/com/google/mediapipe/apps/instantmotiontracking/Sticker.java:35]: Now that you only have one, you can just name it ASSET_3D.

File: mediapipe/examples/android/src/java/com/google/mediapipe/apps/instantmotiontracking/Sticker.java [hunk 5]
```diff
@@ -14,25 +14,35 @@
...
+    /**
+     * Once called, will set the value of the current render to the next
+     * possible Render available. If all possible Renders have been iterated
+     * through, the function will loop and set to the first available Render.
+     */
     public Render iterate() {
       int newEnumIdx = (this.ordinal() + 1) % Render.values().length;
       return Render.values()[newEnumIdx];
-   }
+    }
```
Comment (jianingwei) [inline mediapipe/examples/android/src/java/com/google/mediapipe/apps/instantmotiontracking/Sticker.java:45]: no need for extra space here.

File: mediapipe/examples/android/src/java/com/google/mediapipe/apps/instantmotiontracking/Sticker.java [hunk 6]
```diff
@@ -41,99 +51,138 @@ public Render iterate() {
...
   private static int globalIDLimit = 1;
 
+  /**
+   * Used to create a Sticker object with a newly generated stickerId and a
+   * default Render of the first possible render in our Render enum.
+   */
   public Sticker() {
-    // Every sticker will have a default render of the robot animation
-    this.currentRender = Render.ROBOT;
+    // Every sticker will have a default render of the first non-GIF asset
```
Comment (jianingwei) [inline mediapipe/examples/android/src/java/com/google/mediapipe/apps/instantmotiontracking/Sticker.java:73]: 3D asset

**Pull request merged**